src/sentry/integrations/issues.py

```diff
@@ -51,3 +67,45 @@ def get_create_issue_config(self, group, **kwargs):
                 'type': 'textarea',
             }
         ]
+
+    def message_from_error(self, exc):
```

Comment:
this is copied from sentry-plugins

Reply:
Why's this part of the `IssueSyncMixin`?